ext/spl/spl_array.stub.php

```diff
@@ -280,7 +280,7 @@ public function count(): int {}
     public function rewind(): void {}
     public function valid(): bool {}
 
-    /** @return mixed */
+    /** @return int|string */
     public function key() {}
```

Comment:
I meant something else:

```suggestion
    public function key(): int|string {}
```

The arginfo.h files didn't change at all - union types are parsed from the signature, not comments - comments are more for unrepresentable types like `resource`

```
class ArrayIterator implements SeekableIterator, ArrayAccess, Serializable, Countable
{
    public function __construct(array|object $array = [], int $flags = 0) {}
```

Reply:
Ah, oops. I knew what you meant, I just did the wrong thing in a hurry.